### DISCUSSION TITLE
Add New Attributes (Columns) to Lists in Discover Assets

### DISCUSSION THREAD

**Discussion by adoc-github** [QualiTorque/qualitorque.github.io#174] [Ideas]
![Discover Assets](https://user-images.githubusercontent.com/65012009/205198421-b656a1cd-ccde-4068-860a-07ebbe58a7d2.png)

This is a request from our Customers.
Please add attributes to the Discover Assets list in the image above, such as the provider to use and the resource to be created.
(Example resource attributes are [aws_instance, azurerm_virtual_machine], etc.)
When creating a Blueprint, open and examine all IaC modules in the repository, rename them to have rules for naming modules,
Selecting IaC modules in Assets takes a lot of time.

But if users can see attribute information from Torque's Discover Assets list, the value to the product goes up a lot.
do you like this idea?
Also, to realize this function, it is necessary to read the contents of IaC from Torque.
Is it technically possible?

**Comment by TomerAdmon** [comment]
Hi @adoc-github, 
Thank you for sharing this idea with us 🙏
We will take it with the product team and see if we can prioritize it for future iterations.

**Reply by adoc-github** [reply]
Hi @TomerAdmon,
Any progress on considering implementing this idea for Torque?
Will it be added in future Torque updates?
I would like to know the status of the consideration.

**Reply by adoc-github** [reply]
Hi @TomerAdmon, 
Have you decided anything about prioritizing this idea?

**Reply by shirabendor-quali** [reply]
Hi @adoc-github ,
At this point we are not going to implement this feature request, as we're considering a change in repo discovery altogether. 
We hope the new design will fit your needs.
